### DISCUSSION TITLE
build master branch - '.' is not recognized as an internal or external command,

### DISCUSSION THREAD

**Discussion by orenmiron** [google/model-viewer#3277] [Q&A]
hi
I pulled the git rpository and tried to follow the build instructions.
the build failed. see PowerShell console below
thanks.

Windows PowerShell
Copyright (C) Microsoft Corporation. All rights reserved.

Try the new cross-platform PowerShell https://aka.ms/pscore6

PS C:\Windows\system32> d:
PS D:\> cd .\Development\model-viewer\model-viewer-master\
PS D:\Development\model-viewer\model-viewer-master> npm run bootstrap

> bootstrap
> ./node_modules/.bin/lerna bootstrap --ci

**'.' is not recognized as an internal or external command,**
operable program or batch file.
PS D:\Development\model-viewer\model-viewer-master> npm run build

> build
> ./node_modules/.bin/lerna run build --stream

**'.' is not recognized as an internal or external command,**
operable program or batch file.
PS D:\Development\model-viewer\model-viewer-master>

**Comment by elalish** [comment]
Please follow the windows setup instructions on the front page README. You'll need to use WSL2 instead of PowerShell.
